doc/Options/z390_options.md

```diff
@@ -0,0 +1,129 @@
+MAXQUE(1000) | E | 1000 | Maximum output queue length from any CMD started task before queue messages are automatically copied to GUI output log to conserve memory and prevent stall.
+MAXRLD(10000) | L | 10000 | Maximum RLD relocation records allowed in a program.
+MAXSIZE(50)| MALE | 50 | Maximum size of any output file in MB
+MAXSYM(50000) | MA | 50000 | Maximum symbols in one macro assembly
+MAXWARN(4) | MA | 4 | Maximum MNOTE warning level without generating error.
+MAXWIDTH(800) | E | 800 | Maximum pixel width for GUI dialog windows
+MCALL | A | NO | List each macro call and exit on the PRN assembly listing in addition to first level calls if LISTCALL option is also on.
+MEM(1) | E | 1 | Memory allocated for 390 program execution in MB.  Note for RMODE31 loads and GETMAIN's memory above the 16 MB line must be allocated.  For example MEM(32) would provide 16 MB below the line and 16 MB above.
+MINHEIGHT(150) | E | 150 | Minimum pixel height for GUI dialog windows
+MINWIDTH(150) | E | 150 | Minimum pixel width for GUI dialog windows
+MNOTE(0) | MA | 0 | Control MNOTE error/warning messages during macro expansion and assembly: <br/> 0 - default generates MNOTE error/warning during mz390 and az390 <br/> 1 - suppress mz390 error/warning and only pass MNOTE to az390 <br/> 2 - generates MNOTE error/warning during mz390 and suppresses passing them on to az390
+MOD | L | NO | Generate raw code file from lz390 with .MOD suffix and no header or trailer and no RLD's. 
+OBJ | A | YES | Generate relocatable object code file
+OBJHEX | A | NO | Generate ASCII readable form of OBJ file with support for 31 bit long sections.  The default is to generate binary OBJ file which is compatible with mainframe linkers but is limited to 24 bit long sections.
+PARM('text') | E | none | Define character string parm to passed to executable program via address in R1 at startup pointing to a fullword pointing to a halfword length followed by EBCDIC characters. When the length is zero, no PARM is present.
+PC | M | YES | Generate macro pseudo code in cache memory for speeding up macro code that is executed more than once during macro expansion.
+PCOPT | M | YES | Optimize macro pseudo code by looking for push, add/sub, store sequences and replacing them with inc/dec pseudo code opcodes.  These pseudo codes are maintained in a cache in memory during macro execution only and have no affect on generated object code.
+PRINTALL | A | NO | Suppress PRINT OFF and PRINT NOGEN commands to force all source lines on PRN listing.
+PDSMEM8 | M | NO | If PDSMEM8 option is specified, error messages will be issued for any macro or copybook file names with length greater than 8.  This options is available to provide compatibility with mainframe PDS libraries that are limited to 8 character names. Note zCOBOL and zCICS use names longer than 8 characters such as zcobol\IDENTIFICATION.MAC and cics\CICS_INQUIRE.MAC.
```

Comment:
Line 78. Second sentence. "This options is ..." -> "This option is ...".

Reply:
Fixed 40c73aa